src/Grpc.AspNetCore.HealthChecks/Grpc.AspNetCore.HealthChecks.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I was wondering whether you would notice that.

gRPC calls it Grpc.HealthCheck while ASP.NET Core calls it HealthChecks. I went with HealthChecks, just because I assume that is what ASP.NET Core developers are more familiar with.

Reply:
sg